src/components/input/demoBasicUsage/index.html

```diff
@@ -83,7 +83,7 @@
 
         <md-input-container class="md-block">
           <label>Biography</label>
-          <textarea ng-model="user.biography" columns="1" md-maxlength="150" rows="5"></textarea>
+          <textarea ng-model="user.biography" md-maxlength="150" rows="5" md-select-on-focus></textarea>
```

Comment:
@ThomasBurleson - Done. 

PS: `columns` seems not be a valid attribute for a native textarea, also for the `md <textarea>`.